stock_available_mrp/models/product_product.py

```diff
@@ -144,6 +144,9 @@ def _get_components_needs(self, exploded_components):
         needs = Counter()
         for bom_component in exploded_components:
             component = bom_component[0].product_id
-            needs += Counter({component: bom_component[1]['qty']})
+            if only_stockable and component.type != "product": #ignore 'service' and 'consu' in BoM to set potential_qty=-1.0
```

Comment:
move the comment to line above the `if` statement, and check that the line length is < 80char